designer/server/src/test/scala/pl/touk/nussknacker/test/utils/domain/ReflectionBasedUtils.scala

```diff
@@ -0,0 +1,23 @@
+package pl.touk.nussknacker.test.utils.domain
+
+import org.reflections.Reflections
+import org.reflections.util.ConfigurationBuilder
+
+import scala.jdk.CollectionConverters._
+
+object ReflectionBasedUtils {
```

Comment:
It would be nice if you could use it in `NuDesignerApiAvailableToExpose#findApiEndpointsClasses` too

Reply:
Done :)